### PR TITLE
Fixed recursion in fetcher, added default spm8 environ for out of the box usable install

### LINKED ISSUE #195: Max recursion reached when trying to download spm auditory data
```
 File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/pypreprocess/datasets.py", line 111, in fetch_spm_auditory
    subject_id=subject_id)
  File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/pypreprocess/datasets.py", line 111, in fetch_spm_auditory
    subject_id=subject_id)
  File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/pypreprocess/datasets.py", line 111, in fetch_spm_auditory
    subject_id=subject_id)
  File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/pypreprocess/datasets.py", line 105, in fetch_spm_auditory
    _fetch_file(url, subject_dir)
  File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/nilearn/datasets/utils.py", line 463, in _fetch_file
    parse = _urllib.parse.urlparse(url)
  File "/opt/anaconda/envs/py27/lib/python2.7/site-packages/nilearn/_utils/compat.py", line 45, in __getattr__
    if not isinstance(attr, types.ModuleType):
RuntimeError: maximum recursion depth exceeded while calling a Python object
```

Related to nistats/nistats#61

### PR DESCRIPTION
**Pull request by arthurmensch**
New paths in `configure_spm` permits a working install after running `continuous_intergration/install_spm.sh`.

Fixes #195 

I wonder whether we should not embed the spm dir within pypreprocess directory -- not critical though.

### CODE REVIEW

**Comment by dohmatob** [comment]
Oops! Looks like we need to rebase this one upon master.

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,7 @@
 *.o
 *.so
 *.a
+
+.idea/
+
+.coverage
```
Comment (dohmatob) [inline .gitignore]: Missing newline.

**Comment by dohmatob** [comment]
I merge this once rebased and new line added.

**Comment by arthurmensch** [comment]
Done

**Pull request merged**